mkosi

```diff
@@ -1338,6 +1338,13 @@ def install_debian_or_ubuntu(args, workspace, run_build_script, mirror):
         f.write("#!/bin/sh\n")
         f.write("exit 101")
     os.chmod(policyrcd, 0o755)
+    # Work around "Failed to find module 'crc32c'" dracut issue
+    # See also:
+    # - https://github.com/antonio-petricca/buddy-linux/issues/2#issuecomment-404505527
+    # - https://bugs.launchpad.net/ubuntu/+source/dracut/+bug/1781143
```

Comment:
It might be nice to include these comments in the `dracut_bug_conf` file as well, but I’m not sure how important that is.